From 102082f3c245270a597ba5c55d0fb07217133bf3 Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Thu, 26 Jan 2017 13:50:39 +0800
Subject: [PATCH] fs/yaffs2: adjust to the change of inode_change_ok()

In commit cb8e1eef351b ("fs: Give dentry to inode_change_ok() instead
of inode") the inode_change_ok() has been rename to setattr_prepare(),
and one of the argument was also changed from inode to dentry. We
adjust the invocation of inode_change_ok() in Yaffs2 according to
this change.

Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 fs/yaffs2/yaffs_vfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/yaffs2/yaffs_vfs.c b/fs/yaffs2/yaffs_vfs.c
index 5a9c295aecc8..e5e0c650baca 100644
--- a/fs/yaffs2/yaffs_vfs.c
+++ b/fs/yaffs2/yaffs_vfs.c
@@ -885,7 +885,7 @@ static int yaffs_setattr(struct dentry *dentry, struct iattr *attr)
 #endif
 
 	if (error == 0)
-		error = inode_change_ok(inode, attr);
+		error = setattr_prepare(dentry, attr);
 	if (error == 0) {
 		int result;
 		if (!error) {
-- 
2.5.0

